glossary.yml

```diff
@@ -4436,13 +4442,20 @@
     term: "false positive"
     def: >
       Data points which are actually [false](#false) but incorrectly predicted as [true](#true).
+
+  pt:
+    term: "falso positivo"
+    def: >
+      Dados ou resultados que na realidade são [falso](#false) mas que são incorretamente considerados ou determinados como [verdadeiro](#true) pelo algoritmo.
```

Comment:
Suggest the following : Dados ou resultados que são realmente falsos, mas são incorretamente previstos como verdadeiros pelo algoritmo